packages/sirius-web/backend/sirius-web/src/test/resources/scripts/initializeMigrationParticipant.sql

```diff
@@ -0,0 +1,52 @@
+-- Sample studio project
+INSERT INTO project (
+  id,
+  name,
+  created_on,
+  last_modified_on
+) VALUES (
+  '01234836-0902-418a-900a-4c0afd20323e',
+  'Studio',
+  '2024-01-01 9:42:0.000',
+  '2024-01-02 9:42:0.000'
+);
+INSERT INTO nature (
+  project_id,
+  name
+) VALUES (
+  '01234836-0902-418a-900a-4c0afd20323e',
+  'siriusComponents://nature?kind=studio'
+);
+INSERT INTO semantic_data (
+  id,
+  project_id,
+  created_on,
+  last_modified_on
+) VALUES (
+  'e344d967-a639-4f6c-9c00-a466d51063c6',
+  '01234836-0902-418a-900a-4c0afd20323e',
+  '2024-01-01 9:42:0.000',
+  '2024-01-02 9:42:0.000'
+);
+INSERT INTO semantic_data_domain (
+  semantic_data_id,
+  uri
+) VALUES (
+  'e344d967-a639-4f6c-9c00-a466d51063c6',
+  'http://www.eclipse.org/sirius-web/view'
+);
+INSERT INTO document (
+  id,
+  semantic_data_id,
+  name,
+  content,
+  created_on,
+  last_modified_on
+) VALUES (
+  'ed2a5355-991d-458f-87f1-ea3a18b1f104',
+  'e344d967-a639-4f6c-9c00-a466d51063c6',
+  'Flow View',
+  '{"json":{"version":"1.0","encoding":"utf-8"},"ns":{"diagram":"http://www.eclipse.org/sirius-web/diagram","view":"http://www.eclipse.org/sirius-web/view"},"content":[{"id":"9674f8f7-ff1a-4061-bb32-a4a235a9c2ca","eClass":"view:View","data":{"descriptions":[{"id":"22fb1f4d-109d-4e73-bff0-f7cd96fb5fbb","eClass":"diagram:DiagramDescription","data":{"name":"Diagram Description","domainType":"flow::System","nodeDescriptionsWrong":[{"id":"6949ddfe-f480-473b-bc8a-6f2bdde07e4d","eClass":"diagram:NodeDescription","data":{"name":"NodeWithoutImage","domainTypeWrong":"flow::CompositeProcessor","labelExpression":"aql:''NodeWithoutImage''","childrenLayoutStrategy":{"id":"20651d93-2ee5-41cb-b2bd-1e75958c73cf","eClass":"diagram:FreeFormLayoutStrategyDescription"},"style":{"id":"92fe9d3f-2c5b-41ab-81ea-8482c8cd57b9","eClass":"diagram:RectangularNodeStyleDescription","data":{"color":"view:FixedColor 1952d117-7d88-32c4-a839-3858e5e779ae#//@colorPalettes.1/@colors.1","borderColor":"view:FixedColor 1952d117-7d88-32c4-a839-3858e5e779ae#//@colorPalettes.1/@colors.0","labelColor":"view:FixedColor 1952d117-7d88-32c4-a839-3858e5e779ae#//@colorPalettes.1/@colors.0"}}}},{"id":"88f95390-ac15-4381-ae82-7b23a2017bd4","eClass":"diagram:NodeDescription","data":{"name":"NodeWithImage","domainTypeWrong":"flow::CompositeProcessor","labelExpression":"aql:''NodeWithImage''","childrenLayoutStrategy":{"id":"db92e810-5d51-4d0c-a6cf-6a1e00cea6d9","eClass":"diagram:FreeFormLayoutStrategyDescription"},"style":{"id":"fa35806a-bdd0-4ba7-81bf-8cfba8f8fef5","eClass":"diagram:ImageNodeStyleDescription","data":{"color":"view:FixedColor 1952d117-7d88-32c4-a839-3858e5e779ae#//@colorPalettes.1/@colors.1","borderColor":"view:FixedColor 1952d117-7d88-32c4-a839-3858e5e779ae#//@colorPalettes.1/@colors.0","labelColor":"view:FixedColor 1952d117-7d88-32c4-a839-3858e5e779ae#//@colorPalettes.1/@colors.0","shape":"ae1a67e3-d455-3198-9f35-372ef2a3006d"}}}}]}}]}}]}',
```

Comment:
`nodeDescriptionsWrong` don't create fake data, use small real migration examples with a proper naming convention to figure out what you are testing.
For example, name this document something like `NodeDescription#labelExpression migration` etc. This file will grow big very rapidly. I don't need here useless data, I only need a View, a diagram description and a node description nothing more. I don't need colors, styles, imaginary examples.